.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4.2.2
```

Comment:
Why do you use specific version, not just v4 ?

Reply:
no particular reason - usually I just explicitly grab the latest when updating. Do github actions work with semantic versioning - aka, doing `@v4` will grab latest available?

Reply:
yup, just confirmed that they seem to be re-tagging repo